HowToCompile.txt

```diff
@@ -69,7 +69,7 @@ cd ffmpeg
 make
 Copy *.lib in Project\MSVC2013\GUI\Lib
 Mac:
-./configure --enable-gpl --enable-version3 --enable-shared --disable-swscale-alpha --disable-doc --disable-ffplay --disable-ffprobe --disable-ffserver --disable-debug --enable-libopenjpeg --disable-decoder=jpeg2000 --yasmexe=../yasm/yasm --extra-cflags="-I../openjpeg/usr/include" --extra-ldflags="-L../openjpeg/usr/lib -mmacosx-version-min=10.5"
+./configure --enable-gpl --enable-version3 --enable-shared --disable-swscale-alpha --disable-doc --disable-ffplay --disable-ffprobe --disable-ffserver --disable-debug --enable-libopenjpeg --disable-decoder=jpeg2000 --yasmexe=../yasm/yasm --extra-cflags="-I../openjpeg/usr/include" --extra-ldflags="-L../openjpeg/usr/lib -mmacosx-version-min=10.8"
```

Comment:
>`--disable-ffserver`

Do you still have `ffserver` in the source code?

Reply:
no